src/colorpickr.js

```diff
@@ -124,87 +137,86 @@ class ColorPickr extends React.Component {
     const hsl = rgb2hsl(r, g, b);
     const hex = rgb2hex(r, g, b);
 
-    const nextColor = Object.assign({}, color, j, hsl, { hex });
+    const nextColor = { ...color, ...j, ...hsl, ...{ hex } };
```

Comment:
This spread operator followed by a set state is repeated a lot: could be a shared method.